docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/snowflake.md

```diff
@@ -0,0 +1,301 @@
+---
+sidebar_position: 2
+---
+
+# Snowflake Destination
+
+In order to create an end to end test of the ELT pipeline apart from the Source which is the KYVE network, you should
```

Comment:
I'd say a bit unclear -- perhaps can cut the whole sentence or put it after the next sentence (which introduces this page well I think)?